bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/ModuleType.java

```diff
@@ -79,75 +78,77 @@
 public enum ModuleType {
     UNKNOWN(FeatureArea.NONE, "", null, List.of(), List.of()),
     ACCOUNT(FeatureArea.NONE, "", null, List.of(), List.of()),
-    @SerializedName("NAHome")
+
     HOME(FeatureArea.NONE, "NAHome", ACCOUNT,
             List.of(DeviceCapability.class, EventCapability.class, HomeCapability.class, ChannelHelperCapability.class),
             List.of(HomeSecurityChannelHelper.class, HomeEnergyChannelHelper.class)),
-    @SerializedName("NAPerson")
+
     PERSON(FeatureArea.SECURITY, "NAPerson", HOME,
             List.of(EventCapability.class, PersonCapability.class, ChannelHelperCapability.class),
             List.of(PersonChannelHelper.class, EventPersonChannelHelper.class)),
-    @SerializedName("NACamera")
+
     WELCOME(FeatureArea.SECURITY, "NACamera", HOME,
             List.of(EventCapability.class, CameraCapability.class, ChannelHelperCapability.class),
             List.of(CameraChannelHelper.class, SignalChannelHelper.class, EventChannelHelper.class)),
-    @SerializedName("NOC")
+
+    SIREN(FeatureArea.SECURITY, "NIS", WELCOME, List.of(ChannelHelperCapability.class),
```

Comment:
Not fair for the user !

Reply:
Not fair ? I'm not sure I got your comment. The alarm reacts to events generated by the camera (unknown face...) so I understand why it could not work without. When you pair it, you pair it with the camera. Unfair, but this is how it works.

Reply:
Ok, my fault, I don't know why, but I was thinking about a smoke detector rather than a siren !
So yes, it makes sense to attach it to something that detects alarm.